packages/workbox-build/src/entry-points/get-manifest.js

```diff
@@ -0,0 +1,26 @@
+const GetManifestOptions = require('./options/get-manifest-options');
+const getFileManifestEntries = require('../lib/get-file-manifest-entries');
+
+/**
+ * This method returns a list of URLs to precache, referred to as a "precache
+ * manifest", along with details about the number of entries and their size,
+ * based on the options you provide.
+ *
+ * @param {Object} input
+ * @return {Promise<{manifestEntries: Array<ManifestEntry>,
+ * count: Number, size: Number}>} A promise that resolves once the precache
+ * manifest is determined. The `size` property contains the aggregate size of
+ * all the precached entries, in bytes, the `count` property contains the total
+ * number of precached entries, and the `manifestEntries` property contains all
+ * the `ManifestEntry` items.
+ *
+ * @memberof module:workbox-build
+ */
+async function getManifest(input) {
```

Comment:
I've removed the old [`generateFileManifest()`](https://workboxjs.org/reference-docs/latest/module-workbox-build.html#.generateFileManifest) method in favor of this. It runs the same logic, but instead of writing the results to disk (and making the user choose between CJS and ESM format), it just returns the manifest as a variable, for added flexibility.

Folks who want the old behavior can still serialize that variable to disk in the format they prefer.